src/applications/disability-benefits/all-claims/utils.tsx

```diff
@@ -4,15 +4,29 @@ import moment from 'moment';
 import get from '../../../platform/utilities/data/get';
 
 import { RESERVE_GUARD_TYPES } from './constants';
+
+interface DateRangeType {
+  from: string;
+  to: string;
+}
+
+interface FormDataType {
+  servicePeriods: Array<{
+    serviceBranch: string;
+    serviceHistory: Array<{}>;
+    dateRange: DateRangeType;
+  }>;
+}
+
+
 /**
  * Show one thing, have a screen reader say another.
  * NOTE: This will cause React to get angry if used in a <p> because the DOM is "invalid."
  *
- * @param {ReactElement|ReactComponent|String} srIgnored -- Thing to be displayed visually,
- *                                                           but ignored by screen readers
- * @param {String} substitutionText -- Text for screen readers to say instead of srIgnored
+ * @param srIgnored -- Thing to be displayed visually, but ignored by screen readers
+ * @param substitutionText -- Text for screen readers to say instead of srIgnored
  */
-const srSubstitute = (srIgnored, substitutionText) => {
+const srSubstitute = (srIgnored: React.ReactNode, substitutionText: string): React.ReactNode => {
```

Comment:
If anybody can find documentation for the React types, that'd be awesome. Like I said in the PR description, Mark found a thing that happened to mention this type. I looked in `node_modules/@types/react/index.d.ts` and found it.